proto/src/constraints.proto

```diff
@@ -31,9 +32,22 @@ message SummaryConstraintMsg {
   oneof second {
      string second_field = 3;
      double value = 4;
+     SummaryBetweenConstraintMsg between = 5;
   }
-  Op op = 5;
-  bool verbose = 6;
+  Op op = 6;
+  bool verbose = 7;
```

Comment:
This will be a breaking change if we change the codes for protobuf files because previous versions serialized will have the old codes. In this case I think its ok because we shouldn't have existing profiles with constraints since they are new, but we need to be careful of changes to protobuf files: adding things is ok, but need to preserve the existing codes. So lets use '7' for the between field and then leave 'op' and 'verbose' unchanged?

Reply:
Yes, I think it wouldn't be a problem.